### PR TITLE
Feature/cards and boxes using bslib

### PR DESCRIPTION
**Pull request by aj-thl**
Implemented cards and boxes by using bslib.

### CODE REVIEW

**Comment by tinneuro** [comment]
Great thanks a lot I will do the review today or tomorrow :)

File: NEWS.md [hunk 1]
```diff
@@ -26,6 +26,8 @@
 * Improved wording of objects in signals and reports tab
 * Made table appearance for interactive and static tables consistent using new dependency `flextable`
 * Recodes NAs in signal tables to 'unknown' for all stratifications but not for the unstratified level
+* Changed the title of the tool to "Signal Detection Tool".
```
Comment (tinneuro) [inline NEWS.md:29]: Can you add these news to the "new section 0.2.1"? I added now in develop because 0.2.0 was already published

**Comment by tinneuro** [comment]
Furthermore shinyWidgets needs to be added to the DESCRIPTION Imports

**Comment by tinneuro** [comment]
I really like the new style!! It is great. Thanks a lot for your work!

**Comment by aj-thl** [comment]
Thanks @tinneuro! It's nice to hear that you like the changes 😄 I updated the NEWS and the DESCRIPTION as you requested. I also noticed that pr was merging into main instead of the develop branch so I also corrected this by changing the base branch from main to develop. You may need to do the review again, my apologies for this.

**Comment by tinneuro** [comment]
Great no worries. I was anyways not finished yet

**Comment by tinneuro** [comment]
I found one bug: when using place_of_infection as strata then we get an error and nothing is shown. Maybe you can shortly investigate this?

File: NEWS.md [hunk 2]
```diff
@@ -27,9 +32,6 @@
 * Allow selection of all character and factor variables in the dataset, except for the variables ending with '_id', to use for filters and stratifications.
 * Adding a function `pad_signals()` performing the padding of signals timeseries (extension of threshold and expecation to the past for visualisation) which was previously done in the signals tab server.
 * adding method and number_of_weeks to the resulting tibble of `get_signals()` to reduce number of parameters needed to be passed forward.
-* Improved wording of objects in signals and reports tab
```
Comment (tinneuro) [inline NEWS.md:30]: Probably just a volatility error: can you move the deleted points back to 0.2.0 and just add your new points to 0.2.1? Thanks

Comment (aj-thl) [reply NEWS.md:30]: Sure!

**Comment by aj-thl** [comment]
What kind of an error you are getting when using place_of_infection as strata? I tried this (other input parameters in default values) and I could not reproduce the error. I put a screenshot of the results below:
image

**Comment by aj-thl** [comment]
The NEWS has now been corrected.

**Comment by tinneuro** [comment]
Somehow the error disappeared which is great :D

**Comment by tinneuro** [comment]
I am sorry, I just needed to quickly update something on develop. so now you need to change the NEWS one last time

**Comment by tinneuro** [comment]
I finished reviewing everything so after this resolve I will approve the PR :)

**Review by tinneuro — APPROVED** [review]
Thank you! Great improvments.

**Pull request merged**